src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -395,7 +401,49 @@ private static void ProcessResult(IAsyncResult asyncResult, AsyncCallback asyncC
             HttpInstrumentationEventSource.Log.FailedProcessResult(ex);
         }
 
+        activity.SetEndTime(DateTime.UtcNow);
```

Comment:
what is the need for this change?

Reply:
Cribbed from the .NET/.NET Core implementation that calls out relying on HttpClient setting the end time:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerMetricsDiagnosticListener.cs#L100

Reply:
Don't think its needed here. activity is stopped before we derive metrics from it, so duration is already set.